v2/account_claims.go

```diff
@@ -51,15 +51,16 @@ func (n *NatsLimits) IsUnlimited() bool {
 }
 
 type JetStreamLimits struct {
-	MemoryStorage int64 `json:"mem_storage,omitempty"`  // Max number of bytes stored in memory across all streams. (0 means disabled)
-	DiskStorage   int64 `json:"disk_storage,omitempty"` // Max number of bytes stored on disk across all streams. (0 means disabled)
-	Streams       int64 `json:"streams,omitempty"`      // Max number of streams
-	Consumer      int64 `json:"consumer,omitempty"`     // Max number of consumer
+	MemoryStorage    int64 `json:"mem_storage,omitempty"`        // Max number of bytes stored in memory across all streams. (0 means disabled)
+	DiskStorage      int64 `json:"disk_storage,omitempty"`       // Max number of bytes stored on disk across all streams. (0 means disabled)
+	Streams          int64 `json:"streams,omitempty"`            // Max number of streams
+	Consumer         int64 `json:"consumer,omitempty"`           // Max number of consumer
+	MaxBytesRequired bool  `json:"max_bytes_required,omitempty"` // Max bytes required by Stream
```

Comment:
This setting is effectively informing the server that on JS API requests the stream should provide the max_bytes for the stream. However this setting should be a server configuration setting, because the server is what wants to know if it should reject requests that don't offer this value. Am I understanding this correctly?
@derekcollison

Reply:
we will then need to propagate that setting on all the tooling.

Reply:
Eventually yes but ok for now, just wanted it in for 2.7.